scripts/haas_network.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
+Requires=httpd.service
```

Comment:
> Is it possible to conditionally add a dependency on the database service if it's there? The database could be running elsewhere as well or it could be using sqlite.

I'm pretty sure the `After=` option does what you want. You'll have to explicitly enable the database server, but things should come up in the right order.

https://www.freedesktop.org/software/systemd/man/systemd.unit.html